File: api-sintec/src/main/java/br/com/sintec/model/RetornoFoto.java
package br.com.sintec.model;

import java.time.LocalDateTime;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.JoinColumn;
import javax.persistence.ManyToOne;
import javax.persistence.Table;

import org.springframework.format.annotation.DateTimeFormat;

import com.fasterxml.jackson.annotation.JsonFormat;

import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@ToString
@Getter
@Setter
@Entity
@Table(name="SERV_RETORNO_FOTO")
public class RetornoFoto {

	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	@EqualsAndHashCode.Include
	@Column(name = "ID")
	private Long id;
	@Column(name = "NM_NOME")
	private String nome;
	@Column(name = "DS_PATH")
	private String path;
	@ManyToOne
	@JoinColumn(name="ID_SERVICO")
	private Servico servico;
	@ManyToOne
	@JoinColumn(name="ID_QUESTIONARIO")
	private Questionario questionario;
	@Column(name = "CD_LATITUDE")
	private String latitude;
	@Column(name = "CD_LONGITUDE")
	private String longitude;
	@Column(name="FLAG_ASSINATURA")
	private int flagAssinatura;
	@Column(name="FLAG_RACP")
	private int flagRacp;
	@Column(name = "DT_ATUALIZACAO")
	@DateTimeFormat(pattern = "yyyy-MM-dd HH:mm:ss")
	@JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss", shape = JsonFormat.Shape.STRING)
	private LocalDateTime dataAtualizacao;
	@Column(name="DS_OBSERVACAO")
	private String observacao;
}

File: MOBILE/app_sintec/lib/shared/model/retorno_foto.dart
class RetornoFoto {
  int? id;
  String nome;
  String longitude;
  String latitude;
  String path;
  int? idQuestionario;
  int idServico;
  int flagAssinatura;
  String? observacao;
  String dataAtualizacao;
  int sincronizado;

  RetornoFoto(
      {this.id,
      required this.nome,
      required this.longitude,
      required this.latitude,
      required this.path,
      this.idQuestionario,
      required this.idServico,
      required this.flagAssinatura,
      this.observacao,
      required this.dataAtualizacao,
      required this.sincronizado});

  factory RetornoFoto.fromSQLite(Map map) {
    return RetornoFoto(
      id: map["id"],
      nome: map["nome"],
      longitude: map["longitude"],
      latitude: map["latitude"],
      path: map["path"],
      idQuestionario: map["id_questionario"],
      idServico: map["id_servico"],
      flagAssinatura: map["flag_assinatura"],
      observacao: map["observacao"],
      dataAtualizacao: map['data_atualizacao'],
      sincronizado: map["sincronizado"],
    );
  }

  static List<RetornoFoto> fromSQLiteList(List<Map> listMap) {
    List<RetornoFoto> lista = [];
    for (Map item in listMap) {
      lista.add(RetornoFoto.fromSQLite(item));
    }
    return lista;
  }
}
